openqabot/openqa.py

```diff
@@ -12,6 +12,8 @@
 
 logger = logging.getLogger("bot.openqa")
 
+_DEVELOPMENT_PARENT_GROUP_ID = 9
```

Comment:
I suggest to move that to __init__.py where we have other configuration settings already